chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -494,23 +494,49 @@ mcCombinations.prototype.possibleCombinations = [
 		maxfields: 1,
 		score: 0,
 		autoSubmit: false
+	},
+	{
+		combinationId: 'form001',
```

Comment:
then I'm not sure to follow you: on vk.com i'm actually able to select any login field with the "select custom credential fields" feature.
The goal of this feature is to actually override the complete mccombinations logic.